model/src/main/proto/onboarding.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Structure for user onboarding flow.
+message OnboardingFlow {
+// Determines whether user already onboarded
```

Comment:
Indentation issue, fix this. Also, full stop at the end.

Reply:
Done.